src/test/java/petStoreTests/user/FindUserByUsernameTest.java

```diff
@@ -0,0 +1,58 @@
+package petStoreTests.user;
+
+import client.UserClient;
+import data.DataSet;
+import dto.requests.user.User;
+import io.qameta.allure.Description;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+import petStoreTests.AbstractTest;
+
+import java.io.IOException;
+
+public class FindUserByUsernameTest extends AbstractTest {
+
+    @Description("Verify that we are can find user in the store data base")
+    @Parameters({"User for adding and finding"})
+    @Test(dataProvider = "testData")
+    public void findUserTest(User user) throws IOException {
+
+        LOGGER.info("BEFORE TEST ADD USER");
+        UserClient.postUser(user);
+        LOGGER.info("BEFORE TEST USER ADDED");
+
+        LOGGER.info("START TEST find user in the store data base");
+        User response = UserClient.getUserByUsername(user.getUsername());
+
+        Assert.assertTrue(response.getId() >= 0);
```

Comment:
I don't think this is valid assertion, verifying for non-null value should be enough